presto-main/src/test/java/com/facebook/presto/operator/scalar/TestDateTimeFunctions.java

```diff
@@ -119,8 +120,13 @@ public void testCurrentDateTimezone()
             throws Exception
     {
         TimeZoneKey kievTimeZoneKey = getTimeZoneKey("Europe/Kiev");
-        for (long instant = new DateTime(2000, 6, 15, 0, 0).getMillis(); instant < new DateTime(2016, 6, 15, 0, 0).getMillis(); instant += TimeUnit.HOURS.toMillis(1)) {
+        TimeZoneKey montrealTimeZoneKey = getTimeZoneKey("America/Montreal");
+        // We expect UTC millis later on so we have to use UTC chronology
+        for (long instant = ISOChronology.getInstanceUTC().getDateTimeMillis(2000, 6, 15, 0, 0, 0, 0);
+             instant < ISOChronology.getInstanceUTC().getDateTimeMillis(2016, 6, 15, 0, 0, 0, 0);
+             instant += TimeUnit.HOURS.toMillis(1)) {
```

Comment:
Would our testing be better if we checked not only whole (UTC) hours? Like with `instant += 57 minutes` ?